src/handlers/comment/handlers/index.ts

```diff
@@ -34,30 +35,48 @@ export const userCommands: UserCommands[] = [
     id: IssueCommentCommands.ASSIGN,
     description: "Assign the origin sender to the issue automatically.",
     handler: assign,
+    callback: async (issue_number, comment) => {
+      await addCommentToIssue(comment, issue_number!);
+    },
```

Comment:
This block has been duplicated in the next lines. Please define/implement a callback function and just use it here like the following.

```typescript
    id: IssueCommentCommands.ASSIGN,
    description: "Assign the origin sender to the issue automatically.",
    handler: assign,
    callback: commandCallback
```

you might need to create command callback function somewhere

Reply:
Oh yeah, i got that... refactoring now